extensions/trello/src/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { List, showToast, ToastStyle } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { returnTodos } from "./utils/fetchTodos";
+import { TrelloFetchResponse } from "./trelloResponse.model";
+import { TodoListItem } from "./TrelloListItem";
+
+export default function PackageList() {
+  const [results, setTodos] = useState<TrelloFetchResponse>([]);
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
```suggestion
  const [loading, setLoading] = useState<boolean>(true);
```
Set the initial state to true to completely avoid flicker, forgot that one 😄